### PR TITLE
[RHDEVDOCS-2333] - Update ability to filter helm charts when multiple Helm repos are available

### PR DESCRIPTION
**Pull request by rishumehra**
Updated the information for Chart Repositories filter with its screenshot and a few minor changes.

JIRA issue: [RHDEVDOCS-2333] | URL: https://issues.redhat.com/browse/RHDEVDOCS-2333

Version: OpenShift 4.7

Previous PR: https://github.com/openshift/openshift-docs/pull/28054

### CODE REVIEW

**Comment by Preeticp** [comment]
/lgtm

The Textual changes look good. Please check the rendered version to ensure that the image is rendered properly.

**Review by gajanan-more — APPROVED** [review]
/lgtm

**Review by kalexand-rh — COMMENTED** [review]
I have some suggestions. Please squash your commits.

File: modules/helm-adding-helm-chart-repositories.adoc [hunk 1]
```diff
@@ -43,3 +43,13 @@ EOF
 ----
 +
 . Navigate to  the *Developer Catalog* in the web console to verify that the helm charts from the Azure chart repository are displayed.
```
[the remark this answers is not in the document]
Comment (rishumehra) [reply modules/helm-adding-helm-chart-repositories.adoc:45]: Seems right, we can incorporate this change.

File: modules/helm-adding-helm-chart-repositories.adoc [hunk 2]
```diff
@@ -43,3 +43,13 @@ EOF
...
++
+Use the *Chart repositories* filter to search for the required Helm Chart
++
+.Chart repositories filter
+image::odc_helm_chart_repo_filter.png[]
++
+[NOTE]
+====
+If a cluster administrator removes all of the chart repositories then the user cannot view the Helm option in the *+Add* view, *Developer Catalog*, and left navigation panel.
```
[the remark this answers is not in the document]
Comment (rishumehra) [reply modules/helm-adding-helm-chart-repositories.adoc:54]: That's a great suggestion. Will change it accordingly.

File: modules/helm-adding-helm-chart-repositories.adoc [hunk 3]
```diff
@@ -5,7 +5,7 @@
 [id="adding-helm-chart-repositories_{context}"]
 = Adding custom Helm chart repositories
 
-You can add custom Helm chart repositories to your cluster, and enable access to the Helm charts from these repositories in the *Developer Catalog*.
+As a cluster administrator, you can add custom Helm chart repositories to your cluster, and enable access to the Helm charts from these repositories in the *Developer Catalog*.
```
[the remark this answers is not in the document]
Comment (rishumehra) [reply modules/helm-adding-helm-chart-repositories.adoc:8]: Done

**Comment by rishumehra** [comment]
@kalexand-rh Squashed commits on this PR. Kindly check and merge the PR.

**Pull request merged**